app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -109,7 +110,10 @@ export class PublishRepository extends React.Component<
 
     let selectedIndex = -1
     const selectedOrg = this.props.settings.org
-    for (const [index, org] of this.state.orgs.entries()) {
+    const orgs = [...this.state.orgs.entries()].sort(([, a], [, b]) =>
```

Comment:
💭 Could we do this inside `fetchOrgs` so it only needs to be sorted one, rather than for each `render()`?